kogito-quickstart/pom.xml

```diff
@@ -13,7 +13,7 @@
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus</quarkus.platform.group-id>
     <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
-    <kogito-quarkus.version>1.6.0.Final</kogito-quarkus.version>
+    <kogito-quarkus.version>1.8.0.Final</kogito-quarkus.version>
```

Comment:
Why not 1.9.1.Final? That's the version used for the other quickstarts.

Reply:
there is a bug in that version, I'm working to fix it but most likely we will need a new release and then upgrade all.